students/Devon_Peterka/Lesson_9/Donor.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+class Donor(object):
+    '''
+    Creates an object for a donor containing their first name, last
+    name, and a list of their donations in order.
+    
+    Initialize as: D = Donor(first, last, donation)
+    
+    Donation field is optional.  Multiple initial donations can be input
+    as a list.
+    '''
+    def __init__(self, first, last, donation=[]):
```

Comment:
avoid using mutable objects for default parameters - this can be really dangerous